cmd/skywire-visor/README.md

```diff
@@ -129,182 +129,296 @@ The config is not written to file in the above example
-[2022-02-07T18:49:02-06:00] INFO [transport]: Initialized in 470.128967ms (1.458111681s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [dmsgC]: Connecting to the dmsg network... timeout=20s
-[2022-02-07T18:49:02-06:00] INFO [transport_manager]: transport manager is serving.
-[2022-02-07T18:49:02-06:00] INFO [transport_setup]: Connecting to the dmsg network. local_pk=02da35804dd25dbb91e64e64ca43336206538a775054702d2e543299e5168660ef
-[2022-02-07T18:49:02-06:00] INFO [transport_setup]: Connected! local_pk=02da35804dd25dbb91e64e64ca43336206538a775054702d2e543299e5168660ef
-[2022-02-07T18:49:02-06:00] INFO [transport_setup]: starting listener dmsg_port=47
-[2022-02-07T18:49:02-06:00] INFO [transport_setup]: Accepting dmsg streams. dmsg_port=47
-[2022-02-07T18:49:02-06:00] INFO [transport_manager]: Serving stcp network
-[2022-02-07T18:49:02-06:00] INFO [transport_manager]: listening on network: stcp
-[2022-02-07T18:49:02-06:00] INFO [stcp]: Initialized in 366.457µs (1.458857374s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [transport_setup]: Initialized in 361.602µs (1.458985676s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [transport_manager]: Serving stcpr network
-[2022-02-07T18:49:02-06:00] INFO [transport_manager]: listening on network: stcpr
-[2022-02-07T18:49:02-06:00] INFO [stcp]: listening on addr: [::]:7777
-[2022-02-07T18:49:02-06:00] INFO [public_autoconnect]: Initialized in 9.502µs (1.458628811s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [router]: Starting router
-[2022-02-07T18:49:02-06:00] INFO [dmsgC]: Connected to the dmsg network. timeout=20s
-[2022-02-07T18:49:02-06:00] INFO [transport_manager]: Serving dmsg network
-[2022-02-07T18:49:02-06:00] INFO [transport_manager]: listening on network: dmsg
-[2022-02-07T18:49:02-06:00] INFO [stcpr]: Not binding STCPR: no public IP address found
-[2022-02-07T18:49:02-06:00] INFO [dmsg_ctrl]: Initialized in 696.857µs (1.459423545s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [router]: Initialized in 642.448µs (1.459665206s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [stcpr]: Initialized in 447.316µs (1.459153332s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [public_visor]: Initialized in 787ns (1.459423411s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [launcher]: Initialized in 21.113327ms (1.480061107s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [visor]: Initialized in 226ns (1.502320595s with dependencies)
-[2022-02-07T18:49:02-06:00] INFO [visor]: Initializing hypervisor
-[2022-02-07T18:49:02-06:00] INFO [visor]: Serving RPC client over dmsg. addr=02da35804dd25dbb91e64e64ca43336206538a775054702d2e543299e5168660ef:46
-[2022-02-07T18:49:02-06:00] INFO [visor]: Serving hypervisor... addr=":8000" tls=false
-[2022-02-07T18:49:02-06:00] INFO [visor]: Hypervisor initialized
-[2022-02-07T18:49:02-06:00] INFO [hypervisor]: Initialized in 19.833078ms (1.526147062s with dependencies)
+$ skywire-visor
```

Comment:
I understand that this existed before, but why are we including this amount of sample logs here? What is the user supposed to do with it?